src/templates/item.tpl

```diff
@@ -143,7 +143,7 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 						<label class="control-label" for="category">Category</label>
 						<div class="controls">
 							<select id="category" name="category" class="input-xlarge">
-								<option value="" {if $category == NULL}SELECTED{/if}>Uncategorized</option>
+								<option value="">Uncategorized</option>
```

Comment:
Kinda surprised we weren't letting this become a NULL in the database, but it seems not.

Reply:
You're 100% correct, the schema doesn't make it required; I think I may have fixed the wrong bug here. I was noticing that creating new items would fail silently if category was not set, and I assumed it was because of a `NOT NULL`  in the schema, but the schema allows (and defaults to) `NULL`  for the `items.category` field. 

I will take a look at this today and see if I can work it out.

Reply:
Yeah I'm all for a mandatory category anyway, so I don't mind this rule coming back. It should fix the bug for now.

Reply:
I reverted this change locally, and now I can't make it fail as it was before. "Uncategorized" items save just fine, though with a value of `0` rather than `NULL`. So that's a fun bit of confusion.